src/common/device/AndroidDevice.ts

```diff
@@ -130,14 +146,25 @@ export class AndroidDevice implements BaseDevice {
         return Promise.resolve(result?.trim() ? true : false);
     }
 
+    /**
+     * Attempts to install a native app on the device.
+     *
+     * @param appBundlePath Path to the app bundle of the native app.
+     */
+    public async installApp(appBundlePath: string): Promise<void> {
+        const pathQuote = process.platform === 'win32' ? '"' : "'";
```

Comment:
Nit: We also have this functionality [here](https://github.com/forcedotcom/lwc-dev-mobile-core/blob/2b97e570c2fc8352d81cfc76fcd19826113dc6eb/src/common/AndroidUtils.ts#L723). Perhaps we should define a public method for it in `AndroidUtils`, just to keep it consistent?